src/lib/netlist/core/devices.h

```diff
@@ -21,10 +21,8 @@
 // Namespace starts
 //============================================================
 
-namespace netlist
+namespace netlist::devices
 {
-	namespace devices
-	{
 		// -----------------------------------------------------------------------------
```

Comment:
This header also has the extra level of indent with the nested namespace definition.

Reply:
applied